Oras/Remote/Repository.cs

```diff
@@ -639,7 +639,8 @@ public async Task<Descriptor> GenerateDescriptor(HttpResponseMessage res, Remote
 
 
             // 4. Validate Server Digest (if present)
-            var serverHeaderDigest = res.Content.Headers.GetValues("Docker-Content-Digest");
+            IEnumerable<string> serverHeaderDigest;
+            res.Content.Headers.TryGetValues("Docker-Content-Digest", out serverHeaderDigest);
```

Comment:
`serverHeaderDigest` can still be null, right?

Reply:
Looks like this can be simplified to
```suggestion
            res.Content.Headers.TryGetValues("Docker-Content-Digest", out IEnumerable<string> serverHeaderDigest);
```